gatekeeper/Dockerfile

```diff
@@ -10,15 +10,15 @@ LABEL Name=keycloak-gatekeeper \
       Url=https://github.com/keycloak/keycloak-gatekeeper \
       Help=https://issues.jboss.org/projects/KEYCLOAK
 
-RUN adduser -D -u 1000 gatekeeper
+USER root
```

Comment:
I think some of our users might depend on existing `gatekeeper` user. @abstractj What do you think about this?

Reply:
In the Keycloak container we use the following to add the jboss user, we should do the same to add the gatekeeper user:

echo "jboss:x:1000:jboss" >> /etc/group
echo "jboss:x:1000:1000:JBoss user:/opt/jboss:/sbin/nologin" >> /etc/passwd

Reply:
@slaskawi I think we should keep the images consistent and stick with what @douglaspalmer suggested.

@bruegth would you mind to update your PR?

Reply:
@abstractj I updated my PR.

I added gatekeeper directory under /opt because make it more consistent to Keycloak and I don't like to make /opt as homedir. Also if someone whats to create a local BoltDB relative to workdir, maybe /opt has not the permissions for that.